scintilla/gtk/ScintillaGTK.cxx

```diff
@@ -3250,6 +3250,7 @@ void scintilla_release_resources(void) {
 static void *copy_(void *src) { return src; }
 static void free_(void *doc) { }
 
+GEANY_API_SYMBOL
 GType scnotification_get_type(void) {
 	static gsize type_id = 0;
```

Comment:
I think mentioned this before (maybe on scintilla-interest?), but I don't think there's any point using `gsize` here and keeping casting from `GType` to `gsize`. I think you can just use `GType` everywhere here (or use `gsize` without the casts). Not a big deal, but casts are kind of gross, and could cause subtle bugs if/when `GType` is ever not the same as `gsize`. No biggie though, it's not our code :)